admin/controllers/school.js

```diff
@@ -25,7 +25,8 @@ controller.getSchoolLandingPage = async (req, res, next) => {
     const schoolName = await schoolService.findSchoolByDfeNumber(req.user.School)
     const featureEligibilityData = schoolHomeFeatureEligibilityPresenter.getPresentationData(checkWindowData, req.user.timezone)
     const currentDate = moment.tz(req.user.timezone || config.DEFAULT_TIMEZONE)
-    const isResultsFeatureAccessible = resultPageAvailabilityService.isResultsFeatureAccessible(currentDate, checkWindowData.checkEndDate)
+    const resultsOpeningDay = resultPageAvailabilityService.getResultsOpeningDate(currentDate, checkWindowData.checkEndDate)
+    const isResultsFeatureAccessible = resultPageAvailabilityService.isResultsFeatureAccessible(currentDate, resultsOpeningDay)
```

Comment:
just for readability I am providing here the implementation of isResultsFeatureAccessible method:

```
resultPageAvailabilityService.isResultsFeatureAccessible = (currentDate, resultsAvailabilityDate) => {
  return currentDate.isSameOrAfter(resultsAvailabilityDate)
}
```